crawl-ref/source/ability.cc

```diff
@@ -3941,11 +3941,14 @@ vector<ability_type> get_god_abilities(bool ignore_silence, bool ignore_piety,
             abilities.push_back(static_cast<ability_type>(anc_type));
         }
     }
-    if (silenced(you.pos()) && you_worship(GOD_WU_JIAN) && piety_rank() >= 2)
-        abilities.push_back(ABIL_WU_JIAN_WALLJUMP);
 
     if (!ignore_silence && silenced(you.pos()))
+    {
+        if (you_worship(GOD_WU_JIAN) && piety_rank() >= 2)
```

Comment:
You would probably want to do this like
```suggestion
        if (have_passive(passive_t::wu_jian_wall_jump))
```

Reply:
Nice. DIdn't know about that.